### DISCUSSION TITLE
Why is there such a thick edge under Overview?

### DISCUSSION THREAD

**Discussion by yuancongxri** [gephi/gephi#2822] [Q&A] [answered]
Hi guys, when I import my directed graph dataset, I see a few edges in the overview as shown below. What does this thick edge represent?
![image](https://github.com/gephi/gephi/assets/120722111/409041f1-eaef-4969-b9fd-97f95e1ed612)

**Comment by xBZZZZ** [comment]
edges have different `Weight`

gephi `Graph` tab for some reason draws edge with `Weight`=`1.000001` a lot thicker than edge with `Weight`=`1.0`

**Reply by mbastian** [reply, marked as answer]
Since Gephi 0.11 you can check/uncheck "Rescale Edge Weight" in the visualization settings. By default it's turned on, but you can easy turn it off. You can also set the default to false in Gephi's Preferences. Without rescaling, this should exactly fix your issue.
